src/coreclr/md/compiler/import.cpp

```diff
@@ -1047,8 +1047,8 @@ STDMETHODIMP RegMeta::FindMemberRef(
     LPUTF8              szNameUtf8;
     UTF8STR(szName, szNameUtf8);
 
-    LOG((LOGMD, "MD RegMeta::FindMemberRef(0x%08x, %S, 0x%08x, 0x%08x, 0x%08x)\n",
-        tkPar, MDSTR(szName), pvSigBlob, cbSigBlob, pmr));
+    LOG((LOGMD, "MD RegMeta::FindMemberRef(0x%08x, %s, 0x%08x, 0x%08x, 0x%08x)\n",
+        tkPar, MDSTRA(szNameUtf8), pvSigBlob, cbSigBlob, pmr));
```

Comment:
Why is this using `MDSTRA`, but the other logging statements in this are not?

Reply:
This was done because the `szName` already needed to be converted to UTF8 as opposed to the rest of the locations where converting would be an unnecessary step.

Personally, I would prefer to remove these log message for the APIs. I don't see them having a ton of value nowadays and are just going to make these code paths verbose and annoying audit. Are you okay with me remove the logging of all the arguments? If so, this change would remove even more instances of `%S`.

Reply:
`MDSTRA` is just replacing NULL pointer with `<null>` string. It does not do any conversions.

> Personally, I would prefer to remove these log message for the APIs. 

I would be fine with that.